kafka/status.go

```
@@ -0,0 +1,33 @@
+package kafka
+
+import (
+	"net"
+	"time"
+
+	"github.com/utilitywarehouse/go-pubsub"
+)
+
+func status(brokers []string) (*pubsub.Status, error) {
+	errs := make(chan error)
+
+	for _, broker := range brokers {
+		go func(broker string) {
+			conn, err := net.DialTimeout("tcp", broker, 10*time.Second)
+			if err == nil {
+				err = conn.Close()
+			}
+			errs <- err
```

not sure what you mean? Are you suggesting I wrap like that?

(The conn.Close() error was not returned as a failure in the code I used as reference, but I thought it qualified as a problem that you'd want to know about.)